bindings/profilers/wall.hh

```diff
@@ -65,7 +65,7 @@ class WallProfiler : public Nan::ObjectWrap {
   bool withContexts_ = false;
   bool started_ = false;
   bool workaroundV8Bug_;
-  bool detectV8Bug_;
+  bool detectV8Bug_ = true;
```

Comment:
Do you want to keep this declared for code comprehension purposes? Presumably, the `true` value could just be inlined. Could also become a `constexpr`?

Reply:
That was indeed my intent !
I changed the code to make the member variable `static constexpr`.